pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -380,71 +380,116 @@ func TestModifyUserAccounts(t *testing.T) {
 }
 
 func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
-	// given
 	logf.SetLogger(logf.ZapLogger(true))
 	s := apiScheme(t)
 
-	mur := murtest.NewMasterUserRecord("john",
-		murtest.StatusCondition(toBeNotReady(provisioningReason, "")),
-		murtest.AdditionalAccounts("member2-cluster", "member3-cluster"))
+	t.Run("mur status synced with updated user account statuses", func(t *testing.T) {
+		// given
+		mur := murtest.NewMasterUserRecord("john",
+			murtest.StatusCondition(toBeNotReady(provisioningReason, "")),
+			murtest.AdditionalAccounts("member2-cluster", "member3-cluster"))
+
+		userAccount := uatest.NewUserAccountFromMur(mur,
+			uatest.StatusCondition(toBeNotReady("Provisioning", "")), uatest.ResourceVersion("123abc"))
+		mur.Status.UserAccounts = []toolchainv1alpha1.UserAccountStatusEmbedded{}
+
+		userAccount2 := uatest.NewUserAccountFromMur(mur,
+			uatest.StatusCondition(toBeNotReady("Provisioning", "")), uatest.ResourceVersion("123abc"))
+		mur.Status.UserAccounts = append(mur.Status.UserAccounts, toolchainv1alpha1.UserAccountStatusEmbedded{
+			SyncIndex: "111aaa",
+			Cluster: toolchainv1alpha1.Cluster{
+				Name: "member2-cluster",
+			},
+			UserAccountStatus: userAccount.Status,
```

Comment:
why do you use `userAccount.Status` and not `userAccount2.Status`?

Reply:
because @alexeykazakov didn't write this code - he just moved it to a subtest :-D It was me who wrote it :-) 

technically, it doesn't matter as all the UserAccounts have the same status condition, but I can imagine that it can be a bit confusing

Reply:
```suggestion
			UserAccountStatus: userAccount2.Status,
```